KDIC/BlogUITableExtensionViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  BlogUITableExtensionViewController.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 4/24/16.
+//  Copyright © 2016 Colin Tremblay. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+extension BlogViewController: UITableViewDataSource {
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
```

Comment:
Is this not the default? If it is, you should be able to remove this function completely.